### PR TITLE
Add clearer warnings about potential breakage

### PR DESCRIPTION
**Pull request by Vinnl**
~~I'll still be adding one more string that updates the announcement about tracker removal to clarify how to enable it, but that string hasn't been provided yet.~~ All strings added!

### CODE REVIEW

File: en/app.ftl [hunk 1]
```diff
@@ -388,7 +388,9 @@ setting-api-key-copied = Copied!
 setting-tracker-removal-heading = Remove Email Trackers
 setting-tracker-removal-description = Remove email trackers in all forwarded emails.
 setting-tracker-removal-note = { -brand-name-firefox-relay } can now remove common trackers from emails forwarded through your masks.
+# Deprecated
 setting-tracker-removal-warning = Important: Sometimes removing trackers may cause your email to look broken, because the trackers are often contained within images.
+setting-tracker-removal-warning-2 = Important: Removing trackers may cause your email to look broken, because trackers are often contained in images and links. Any emails you receive like this cannot be fixed or recovered.
```
Comment (Vinnl) [inline en/app.ftl:393]: image

File: en/app.ftl [hunk 2]
```diff
@@ -491,7 +493,9 @@ faq-question-disable-trackerremoval-answer = Yes. If you’re having trouble wit
...
 faq-question-bulk-trackerremoval-answer = You can only turn tracker removal on at an account level — it either removes trackers from all of your emails, or none of them.
 faq-question-trackerremoval-breakage-question = Why do my emails look broken?
+# Deprecated
 faq-question-trackerremoval-breakage-answer = Sometimes removing trackers may cause your email to look broken, because the trackers are often contained within images. When the tracker is removed, the email looks like it’s been formatted wrong because images are missing. This can’t be fixed for emails you’ve already received. If this is preventing you from reading your emails properly, turn off tracker removal.
+faq-question-trackerremoval-breakage-answer-2 = Sometimes removing trackers may cause your email to look broken, because the trackers are often contained within images and links. When the tracker is removed, the email looks like it’s been formatted wrong because images are missing. This can’t be fixed for emails you’ve already received. If this is preventing you from reading your emails properly, turn off tracker removal.
```
Comment (Vinnl) [inline en/app.ftl:498]: image

and also

image

(The latter is an excerpt of the former (the full FAQ), so it makes sense that they're the same strings.)

File: en/app.ftl [hunk 3]
```diff
@@ -568,7 +574,9 @@ profile-stat-label-aliases-used-2 = Email masks used
 # This is displayed in small under a number in a large font indicating the number of trackers that have been removed from all emails sent to all of a user's masks
 profile-stat-label-trackers-removed = Trackers Removed
 profile-stat-label-trackers-learn-more-part1 = Enabling tracker removal will remove common email trackers from your forwarded emails.
+# Deprecated
 profile-stat-label-trackers-learn-more-part2 = Important: Sometimes removing trackers may cause your email to look broken, because the trackers are often contained within images.
+profile-stat-label-trackers-learn-more-part2-2 = Important: Removing trackers may cause your email to look broken, because the trackers are often contained within images and links.
```
Comment (Vinnl) [inline en/app.ftl:579]: image

File: en/app.ftl [hunk 4]
```diff
@@ -552,7 +556,9 @@ profile-label-replies = Replies
 # This is displayed in small under a number in a large font indicating the number of trackers that have been removed from all emails sent to a particular mask
 profile-label-trackers-removed = Trackers Removed
 profile-trackers-removed-tooltip-part1 = With tracker removal enabled, common email trackers will be removed from your forwarded emails.
+# Deprecated
 profile-trackers-removed-tooltip-part2 = Important: Sometimes removing trackers may cause your email to look broken because the trackers are often contained within images.
+profile-trackers-removed-tooltip-part2-2 = <b>Important:</b> Removing trackers may cause your email to look broken, because the trackers are often contained within images and links.
```
Comment (Vinnl) [inline en/app.ftl:561]: image

File: en/app.ftl [hunk 5]
```diff
@@ -851,7 +859,9 @@ trackerreport-removal-explainer-content = { -brand-name-firefox-relay } can now
 trackerreport-trackers-explainer-heading = About email trackers
 trackerreport-trackers-explainer-content-part1 = Email tracking is a common surveillance and advertising tool that has taken over many inboxes. These trackers can be used to understand more about your online behavior, your interests, and your email activity.
 trackerreport-trackers-explainer-content-part2 = A company or organization will embed a tracker in emails sent to you, usually hidden within an image or a link. When the email is opened, code within the tracker sends data back to the company.
+# Deprecated
 trackerreport-breakage-warning = Important: Sometimes removing trackers may cause your email to look broken because the trackers are often contained within images, which will not load if they contain a tracker.
+trackerreport-breakage-warning-2 = Important: Removing trackers may cause your email to look broken, because trackers are often contained in images and links. Any emails you receive like this cannot be fixed or recovered.
```
Comment (Vinnl) [inline en/app.ftl:864]: image

File: en/app.ftl [hunk 6]
```diff
@@ -155,7 +155,9 @@ whatsnew-feature-tracker-removal-heading = Introducing email tracker removal
 # When translating, please make sure the resulting string is of roughly similar
 # length as the English version.
 whatsnew-feature-tracker-removal-snippet = Now { -brand-name-relay } can remove common email trackers from emails forwarded…
+# Deprecated
 whatsnew-feature-tracker-removal-description = Now { -brand-name-relay } can remove common email trackers from emails forwarded to you, helping you stay invisible to advertisers.
+whatsnew-feature-tracker-removal-description-2 = Now { -brand-name-relay } can remove common email trackers from emails forwarded to you, helping you stay invisible to advertisers. Turn it on in “Settings.”
```
Comment (Vinnl) [inline en/app.ftl:160]: image

File: en/app.ftl (hunk 1 above)
Comment (peiying2) [reply en/app.ftl:395]: Didn't catch this in the first go around. Remove the comma before `because`.

File: en/app.ftl [hunk 7]
```diff
@@ -491,7 +495,9 @@ faq-question-disable-trackerremoval-answer = Yes. If you’re having trouble wit
...
 faq-question-bulk-trackerremoval-answer = You can only turn tracker removal on at an account level — it either removes trackers from all of your emails, or none of them.
 faq-question-trackerremoval-breakage-question = Why do my emails look broken?
+# Deprecated
 faq-question-trackerremoval-breakage-answer = Sometimes removing trackers may cause your email to look broken, because the trackers are often contained within images. When the tracker is removed, the email looks like it’s been formatted wrong because images are missing. This can’t be fixed for emails you’ve already received. If this is preventing you from reading your emails properly, turn off tracker removal.
+faq-question-trackerremoval-breakage-answer-2 = Sometimes removing trackers may cause your email to look broken, because the trackers are often contained within images and links. When the tracker is removed, the email looks like it’s been formatted wrong because images are missing. This can’t be fixed for emails you’ve already received. If this is preventing you from reading your emails properly, turn off tracker removal.
```
Comment (peiying2) [inline en/app.ftl:500]: Remove comma before `because`.

File: en/app.ftl [hunk 8]
```diff
@@ -568,7 +576,9 @@ profile-stat-label-aliases-used-2 = Email masks used
 # This is displayed in small under a number in a large font indicating the number of trackers that have been removed from all emails sent to all of a user's masks
 profile-stat-label-trackers-removed = Trackers Removed
 profile-stat-label-trackers-learn-more-part1 = Enabling tracker removal will remove common email trackers from your forwarded emails.
+# Deprecated
 profile-stat-label-trackers-learn-more-part2 = Important: Sometimes removing trackers may cause your email to look broken, because the trackers are often contained within images.
+profile-stat-label-trackers-learn-more-part2-2 = Important: Removing trackers may cause your email to look broken, because the trackers are often contained within images and links.
```
Comment (peiying2) [inline en/app.ftl:581]: Remove comma before `because`.

File: en/app.ftl [hunk 9]
```diff
@@ -851,7 +861,9 @@ trackerreport-removal-explainer-content = { -brand-name-firefox-relay } can now
 trackerreport-trackers-explainer-heading = About email trackers
 trackerreport-trackers-explainer-content-part1 = Email tracking is a common surveillance and advertising tool that has taken over many inboxes. These trackers can be used to understand more about your online behavior, your interests, and your email activity.
 trackerreport-trackers-explainer-content-part2 = A company or organization will embed a tracker in emails sent to you, usually hidden within an image or a link. When the email is opened, code within the tracker sends data back to the company.
+# Deprecated
 trackerreport-breakage-warning = Important: Sometimes removing trackers may cause your email to look broken because the trackers are often contained within images, which will not load if they contain a tracker.
+trackerreport-breakage-warning-2 = Important: Removing trackers may cause your email to look broken, because trackers are often contained in images and links. Any emails you receive like this cannot be fixed or recovered.
```
Comment (peiying2) [inline en/app.ftl:866]: remove comma before `because`

**Review by peiying2 — COMMENTED** [review]
Overall update is fine. The only comment is on the punctuation for all the strings containing `because …`. The comma before it should be removed. Since we are updating the string, let's update this part too.

**Comment by Vinnl** [comment]
Done!

**Review by peiying2 — APPROVED** [review]
LGTM!

**Comment by peiying2** [comment]
@Vinnl this PR can be merged too: https://github.com/mozilla-l10n/fx-private-relay-l10n/pull/97

**Pull request merged**